website/src/components/DataTable/DataTable.tsx

```diff
@@ -198,13 +200,13 @@ const FilterModal = ({
   value: string;
 }) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
-
+  const initialFocusRef = useRef();
```

Comment:
what is the purpose of this ref?

Reply:
@AbdBarho That ref is for auto-focusing on the text input when the filter button is clicked. That way you can start typing as soon as the parent div/button is clicked